client/app/account/api/changePasswordApi.ts

```diff
@@ -0,0 +1,12 @@
+import apiClient from '@/_api/base'
+import { PasswordSchema } from '../schema'
+
+export const changePasswordApi = async (
+  data: PasswordSchema,
```

Comment:
This data type should be like this
`Omit<PasswordSchemam, 'confirmNewPassword'>` same as the argument of mutationFn in useChangePasswordMutation

Reply:
Fixed it.
d2a454df2daf2d0a1857e64502c18f254a7a3d6d